bandit/reporters/progress_base.h

```diff
@@ -94,6 +100,11 @@ namespace bandit {
       }
 
     protected:
+      void update_test_duration() {
+        testcase_duration_  = std::chrono::high_resolution_clock::now() - testcase_start_time_point_;
```

Comment:
This is just again nitpicking about style: can you change the double space before `=` to a single space? (No alignment)

Reply:
Sorry I missed that and will fix it in my next commit. I'm personnaly using [uncrustify](http://uncrustify.sourceforge.net/) to format file automatically and a job to the CI to check files are well formated.

I saw a [clang-format configuration](https://github.com/banditcpp/snowhouse/blob/master/.clang-format) in the latest snowhouse master branch. Adding that to bandit would help!